app/scripts/views/confirm.js

```diff
@@ -49,9 +49,10 @@ define(function (require, exports, module) {
     },
 
     _isOpenGmailButtonVisible: function () {
+      var email = this.getAccount().get('email');
       // The "Open Gmail" is only visible in certain contexts
       // we do not show it in mobile contexts because it performs worse on mobile
-      return this.broker.hasCapability('openGmailButtonVisible');
+      return this.broker.hasCapability('openGmailButtonVisible') && email.indexOf('@gmail.com') > 0;
```

Comment:
I can't believe we didn't think of this earlier. Is it easy enough to pass in the email address instead of fetching it from the account? Super nit, and I won't block on it.